src/game/scenes/arena.c

```diff
@@ -1299,6 +1299,53 @@ void arena_static_tick(scene *scene, int paused) {
     gui_frame_tick(local->game_menu);
 }
 
+void push_players(scene *scene, game_player *p1, game_player *p2) {
```

Comment:
Doxygen-style function comment with a short explanation of what happens here would be nice.